addons/l10n_sg_ubl_pint/models/account_move.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_sg_ubl_pint_uuid = fields.Char(string='Document UUID (SG)', copy=False, help="Universally unique identifier of the Invoice")
```

Comment:
I usually took the habit of putting all attributes that can be seen by the user in double quotes like string and help, feel free to take that suggestion but since you putted double quotes on the help i think it may be relevant :smile: